deploy/lint

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash -e
 
-BASE_DIR="$(dirname "$(dirname "$(readlink -e "$0")")")" # /esp
+# http://stackoverflow.com/a/4774063
+pushd `dirname $0` > /dev/null
```

Comment:
I believe to support spaces you need 
```
pushd "`dirname "$0"`"
```
-- around the `$0` so `dirname` uses the right thing, and around the backticks so the `pushd` does. (Though, if you're using `bash` already, you might consider `$()` instead of backticks.)